src/components/header/header.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyledHeader, StyledContainer, Tel, Logo } from "./styled-header";
+const Header = () => {
+  return (
+    <StyledHeader>
+      <StyledContainer>
+        <Logo>
+          <b>Paint</b>Utah<b>Curbs</b>
+        </Logo>
+        <Tel>TEL: +1(435)-32332-34</Tel>
```

Comment:
who's number is this?

Reply:
I made up this number as a dummy phone number